tests/Sdk/Acceptance/Presentation/Console/Command/Validator/ValidateTaskCommandCest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Sdk\Acceptance\Presentation\Console\Command\Validator;
+
+use PHPUnit\Framework\Assert;
+use SprykerSdk\Sdk\Tests\AcceptanceTester;
+
+/**
+ * @group Sdk
+ * @group Acceptance
+ * @group Presentation
+ * @group Console
+ * @group Command
+ * @group Validator
+ * @group ValidateTaskCommandCest
+ */
+class ValidateTaskCommandCest
+{
+    /**
+     * @var string
+     */
+    protected const COMMAND = 'sdk:validate:task';
+
+    /**
+     * @var string
+     */
+    protected const PROJECT_DIR = 'upgrader_success_project';
+
+    /**
+     * @param \SprykerSdk\Sdk\Tests\AcceptanceTester $I
+     *
+     * @return void
+     */
+    public function testValidateTaskSuccessful(AcceptanceTester $I): void
```

Comment:
Please add tests for the malformed task and task set.

Reply:
it's impossible to do it here.
1. in this case we need to have a broken task definition in sdk.
2. We shouldn't test symfony tool.

Reply:
added